SpringProject/src/main/java/com/gradproject2019/auth/exception/IncorrectPasswordException.java

```diff
@@ -0,0 +1,8 @@
+package com.gradproject2019.auth.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.FORBIDDEN, reason="Password incorrect.")
```

Comment:
This may be a pedantic point, but I question the security of your app based off of the exception messages (if these are the messages displayed on the client)

I could try multiple usernames until I get a "password incorrect" message, and from that I would understand that I have found a user of your application. 

You could fix this by changing the message returned by UserNotFound and IncorrectPassword exceptions (maybe something like "Username or password incorrect"?)